src/django_pg_migration_tools/operations.py

```diff
@@ -1303,6 +1327,22 @@ def _maybe_create_index(self) -> None:
                 unique=False,
             )
 
+    def _maybe_create_unique_constraint(self) -> None:
+        if self.unique is True:
```

Comment:
🐼 
```suggestion
        if self.unique:
```

Reply:
+1 on this

Reply:
"does `self.unique` quack like a boolean?"

3abf9b567b0759199767bb202837b22c535474c9